packages/ace-core/src/l10n/locales/da.json

```diff
@@ -37,22 +37,22 @@
             "testdesc": "Sikrer at '{{name}}' metadata-elementet er til stede"
         },
         "missingpagebreakviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Sidenavigation",
+            "resdesc": "Sikrer at sidelisten indeholder alle sideskift ({{ref}})",
+            "ruledesc": "Publikationens sideliste skal indeholde referencer til at alle sideskift ({{ref}})",
+            "testdesc": "Tjek at alle sideskift er med i sidelisten ({{ref}})"
         },
         "ordertocviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Indholdsfortegnelse",
+            "resdesc": "Rækkefølge for elementer i indholdsfortegnelsen ({{ref}})",
+            "ruledesc": "Indholdsfortegnelse skal være magen til læserækkefølgen ({{ref}})",
+            "testdesc": "Sikrer at publikationens indholdsfortegnelse er i korrekt rækkefølge ({{ref}})"
         },
         "pagelistbrokenviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Sidenavigation",
+            "resdesc": "Sørg for at alle elementer på sidelisten peger på epub:type sideskift ({{ref}})",
```

Comment:
"Sørg for, at alle elementer i sidelisten peger på epub:type sideskift ({{ref}})",